zerver/lib/integrations.py

```diff
@@ -83,6 +83,14 @@ def url_object(self):
     WebhookIntegration('deskdotcom', logo='static/images/integrations/logos/deskcom.png', display_name='Desk.com'),
     WebhookIntegration('freshdesk'),
     WebhookIntegration('github', function='zerver.views.webhooks.github.api_github_landing', display_name='GitHub'),
+    WebhookIntegration(
```

Comment:
It might make sense at this point to have a separate section nearer to the bottom where we put the deprecated versions of things.